library/common/jni/jni_interface.cc

```diff
@@ -92,6 +92,12 @@ extern "C" JNIEXPORT jint JNICALL Java_io_envoyproxy_envoymobile_engine_JniLibra
     JNIEnv* env,
     jclass, // class
     jlong engine, jstring elements, jint count) {
+  const char** tag_arr[2];
```

Comment:
`tag_arr` and `len` are to be passed to `record_counter_inc` as parameters in [upstream change](https://github.com/envoyproxy/envoy-mobile/blob/dc32ac0aeabd42c2f02d0976ee949a03195f7707/library/common/engine.h#L45)

Reply:
Just so I understand: how does a local variable declaration of `tags` result in it being populated by a value from Java? Or is this code incomplete?

Reply:
this pr will have to be integrated with a downstream change to have the tags value populated from java

`jobject tags` is to be passed in from downstream java layer as a `List of Tag(String key, String value)`